pkg/oc/cli/cmd/image/mirror/mirror.go

```diff
@@ -93,6 +90,16 @@ type pushOptions struct {
 	MaxPerRegistry int
 
 	AttemptS3BucketCopy []string
+
+	Filenames []string
```

Comment:
Not adding resource.FilenameOptions as it is unclear whether this command's --filename flag is supposed to have the same behavior as FilenameOptions